hack/update-codecgen.sh

```diff
@@ -31,7 +31,7 @@ generated_files=($(
       \( \
         -wholename './output' \
         -o -wholename './_output' \
-        -o -wholename './staging' \
```

Comment:
@smarterclayton you might want this. Will remove that here again as it is not yet necessary.